src/vabackend.c

```diff
@@ -2062,7 +2062,7 @@ static VAStatus nvExportSurfaceHandle(
 
     drv->backend->fillExportDescriptor(drv, surface, ptr);
 
-    LOG("Exporting with %d %d %d %d %lx %d %d %lx", ptr->width, ptr->height, ptr->layers[0].offset[0], ptr->layers[0].pitch[0], ptr->objects[0].drm_format_modifier, ptr->layers[1].offset[0], ptr->layers[1].pitch[0], ptr->objects[1].drm_format_modifier);
+    LOG("Exporting with %d %d %d %d %llx %d %d %llx", ptr->width, ptr->height, ptr->layers[0].offset[0], ptr->layers[0].pitch[0], (unsigned long long) ptr->objects[0].drm_format_modifier, ptr->layers[1].offset[0], ptr->layers[1].pitch[0], (unsigned long long) ptr->objects[1].drm_format_modifier);
```

Comment:
Doesn't this cause a similar issue on x64? I believe you'll need to use the PRIx64 macro here.